src/main/java/de/abstractolotl/azplace/model/user/UserBotToken.java

```diff
@@ -19,13 +19,16 @@ public class UserBotToken {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    @ManyToOne()
+    @OneToOne()
```

Comment:
Das kann nicht richtig sein.

Reply:
Many To One sollte hier eigentlich schon richtig gewesen sein.
Warum hast du das geändert?

Reply:
Das ist geändert, weil es dann jetzt nur noch einen Bot pro User gibt

Reply:
Aber One To One bedeutet glaub ich es gibt IMMER einen BotToken

Reply:
(und dann kann man gleich die tables zusammen legen)

Reply:
Mit Hibernate ist der default für OneToOne optional=true, dass würde ich jetzt so verstehen, dass der Token auch nicht vorhanden sein kann